js/src/personal/amanda/Types.tsx

```diff
@@ -1,6 +1,8 @@
-interface Project {
+export interface Project {
```

Comment:
nit: After reading about it a bit, I think `type` is likely better than `interface` for us, but both work.